src/NHibernate.Test/Async/SystemTransactions/DistributedSystemTransactionFixture.cs

```diff
@@ -13,13 +13,14 @@
 using System.Threading;
 using System.Transactions;
 using log4net;
-using log4net.Repository.Hierarchy;
 using NHibernate.Cfg;
 using NHibernate.Engine;
-using NHibernate.Linq;
 using NHibernate.Test.TransactionTest;
 using NUnit.Framework;
 
+using sysTran = System.Transactions;
```

Comment:
I dont' like this much.

Reply:
Ok, I will revert.
I did that due to otherwise having to specify the full namespace everywhere before "sysTran" `Transaction` because of the conflict with NHibernate `Transaction` namespace, and I then rather shorten the namespace prefixing.